docs/mine-hnt/validators/troubleshooting.mdx

```diff
@@ -20,24 +20,56 @@ This can be run directly on the Docker container as well:
 
 ```docker exec validator tail -f /var/data/log/console.log```
 
-
 ## Compromised server
 If you find that the server has been compromised just wipe it (save the blocks to avoid having to resync the entire blockchain), close the hole, stand up a new validator and then transfer the stake to it.
 
-
 ## Syncing to the blockchain
 Syncing to the blockchain could be impacted for the following reasons:
 * slow internet connection
 * your validator is disconnecting
 * you just launched an AMI instance (if using AWS)
 
+## Grab the latest blessed snapshot from Helium
+* On a running Validator run `miner snapshot list`
```

Comment:
I would also add a step to verify in the API on the heights available:

https://api.helium.io/v1/snapshots